api-client/src/main/java/com/desk/java/apiclient/model/User.java

```diff
@@ -52,6 +52,7 @@ public void setId(int i) {
         this.id = i;
     }
 
+    @NotNull
```

Comment:
For simple getters like this where there are checks like this I've been changing them to just pass the null property through. I think doing that and leaving it up to the client to change to an empty string for consistency and removing logic from the model objects is a good way to go. What do you think?